tests/v2/validation/rbac/rbac.go

```diff
@@ -0,0 +1,75 @@
+package rbac
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/extensions/projects"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	provisioning "github.com/rancher/rancher/tests/v2/validation/provisioning"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+)
+
+//Helper function to create users
+func createUser(client *rancher.Client)(*management.User,error) {
```

Comment:
We should have some existing function for this isnt it since we have other tests which use standard user to execute? Why are we creating a new helper function? Can we use existing one?